esmvaltool/diag_scripts/hydrology/marrmot.py

```diff
@@ -79,6 +79,20 @@ def _get_extra_info(cube):
     return time_start_end, lat_lon
 
 
+def fix_era5_time_coordinate(cube):
+    """Shift instantaneous variables 30 minutes forward in time.
+
+    After this shift, as an example:
+    time format [1990, 1, 1, 11, 30, 0] will be [1990, 1, 1, 12, 0, 0].
+    For aggregated variables, already time format is [1990, 1, 1, 12, 0, 0].
+    """
+    time = cube.coord(axis='T')
+    time.points = time.points + 1 / 48
```

Comment:
```suggestion
    time.points = time.points + shift / (24 * 60)
```
This will enable an arbitrary value of `shift` in minutes.